browser/js/common/directives/cart/incart.html

```diff
@@ -0,0 +1,21 @@
+<div>
+    <form name="shoppingCart">
+
+        <ul ng-repeat="product in products">
+            <li>
+                {{product.title}}
+            </li>
+            <li >
+                {{product.price | currency }}
+            </li>
+
+            <select ng-model="product.quantity" ng-options="num for num in nums" ng-change="editItem(product)">
```

Comment:
Is there no better way to count things? An index option? Or otherwise?

Maybe this is a better option: https://docs.angularjs.org/api/ng/input/input%5Bnumber%5D